src/components/editor-page/splitter/splitter.tsx

```diff
@@ -130,17 +131,19 @@ export const Splitter: React.FC<SplitterProps> = ({
   return (
     <div ref={splitContainer} className={`flex-fill flex-row d-flex ${additionalContainerClassName || ''}`}>
       <div
-        className={`splitter left ${!showLeft ? 'd-none' : ''}`}
+        {...cypressId('splitter-left')}
+        className={`${styles['splitter']} ${styles['left']} ${!showLeft ? 'd-none' : ''}`}
         style={{ width: `calc(${adjustedRelativeSplitValue}% - 5px)` }}>
         {left}
       </div>
       <ShowIf condition={showLeft && showRight}>
-        <div className='splitter separator'>
+        <div className={`${styles['splitter']} ${styles['separator']}`} {...cypressId('splitter-separator')}>
```

Comment:
We currently only use `split-divider` from `src/components/editor-page/splitter/split-divider/split-divider.tsx` and don't need this…
```suggestion
        <div className={`${styles['splitter']} ${styles['separator']}`}>
```

Reply:
We use that in the cypress tests to simulate the mouse movement.

Reply:
Well we don't. We only ever use it in `editorMode.spec.ts`, which I did not find while searching for this earlier (however that could have happened). But since it is used, I see no reason to remove it.